notebooks/nci_functionality.ipynb

```diff
@@ -0,0 +1,346 @@
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "wf_rest = WaterFacilityRestoration(client)\n",
+    "mapping_set = MappingSet(restorationsvc.get_mapping(wds_restoration_mapping_id))  # new format of mapping\n",
+    "wf_rest.load_remote_input_dataset(\"water_facilities\", \"5a284f2ac7d30d13bc081e52\")  # water facility\n",
+    "wf_rest.set_input_dataset('dfr3_mapping_set', mapping_set)\n",
+    "wf_rest.set_input_dataset(\"damage\", wds_dmg_results)\n",
+    "wf_rest.set_parameter(\"result_name\", \"wf_restoration\")\n",
+    "wf_rest.set_parameter(\"restoration_key\", \"Restoration ID Code\")\n",
+    "wf_rest.set_parameter(\"end_time\", 90.0)\n",
+    "wf_rest.set_parameter(\"time_interval\", 1.0)\n",
+    "wf_rest.run_analysis()\n",
+    "\n",
+    "wds_time_results = wf_rest.get_output_dataset(\"time_results\")\n",
+    "wds_inventory_rest_map = wf_rest.get_output_dataset(\"inventory_restoration_map\")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### EPF/WDS network cascading interdependency analysis"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "nic_func = NciFunctionality(client)\n",
+    "nic_func.set_parameter(\"result_name\", \"MMSA_nci_result\")\n",
+    "nic_func.set_parameter(\"discretized_days\", [1, 3, 7, 30, 90])\n",
+    "nic_func.load_remote_input_dataset(\"epf_network\", epf_mmsa_network)\n",
+    "nic_func.load_remote_input_dataset(\"wds_network\", wds_mmsa_network)\n",
+    "nic_func.load_remote_input_dataset(\"epf_wds_intdp_table\", epf_wds_interdependency)\n",
+    "nic_func.load_remote_input_dataset(\"wds_epf_intdp_table\", wds_epf_interdependency)\n",
+    "nic_func.set_input_dataset(\"epf_subst_failure_results\", epf_subst_failure_results)\n",
+    "nic_func.set_input_dataset(\"epf_inventory_rest_map\", epf_inventory_rest_map)\n",
+    "nic_func.set_input_dataset(\"epf_time_results\", epf_time_results)\n",
+    "nic_func.set_input_dataset(\"wds_dmg_results\", wds_dmg_results)\n",
+    "nic_func.set_input_dataset(\"wds_inventory_rest_map\", wds_inventory_rest_map)\n",
+    "nic_func.set_input_dataset(\"wds_time_results\", wds_time_results)\n",
+    "nic_func.run_analysis()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Obtaining NCI results"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "result = nic_func..get_output_dataset(\"epfDiscretizedCascadingFunc\")\n",
```

Comment:
Typo, two .. and this should be:

result = nic_func.get_output_dataset("epf_cascading_functionality")

Reply:
Fixed